src/MoBi.Presentation/DTO/CreateModuleDTO.cs

```diff
@@ -17,7 +17,7 @@ public CreateModuleDTO()
 
       public void AddForbiddenNames(IReadOnlyList<string> prohibitedNames)
       {
-         _prohibitedNames = prohibitedNames;
+         _prohibitedNames = prohibitedNames.Select(x => x.Trim().ToLower()).ToList();
```

Comment:
Trim().ToLower(). Create a method so that you don't repeat this logic twice. But I am sure it's already done

Reply:
ObjectBaseDTO